src/ListContainer.test.jsx

```diff
@@ -0,0 +1,29 @@
+import { render, fireEvent } from '@testing-library/react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import ListContainer from './ListContainer';
+
+import { tasks } from '../fixtures/tasks';
+import { deleteTask } from './action';
```

Comment:
```suggestion
import { deleteTask } from './action';

import { tasks } from '../fixtures/tasks';
```

fixture와 액션 임포트를 구분해주면 좋을 것 같아요